stable/artifactory/values.yaml

```diff
@@ -1290,3 +1290,12 @@ filebeat:
     output:
       logstash:
          hosts: ["{{ .Values.filebeat.logstashUrl }}"]
+
+# FLUENT BIT SIDECAR CONFIGURATION
+fluentd:
```

Comment:
`fluentd` is different than `fluent-bit`. I think it's confusing.
IMO we should call this `fluent-bit`.

Reply:
will do

Reply:
done